setup.cfg

```diff
@@ -1,4 +1,13 @@
+[coverage:run]
+source = openslides
+
+[coverage:html]
+directory = development/htmlcov
```

Comment:
Ok, we can split up the directory with settings and database on the hand and some tmp files on the other. Maybe we should rename these dirs.
What about `var` for settings and database ... and `tmp` for htmlcov ...?